Documentation/AdvancedBuildInstructions.md

```diff
@@ -42,6 +42,7 @@ directory to `Build/i686` and then running `ninja <target>`:
 
 There are some optional features that can be enabled during compilation that are intended to help with specific types of development work or introduce experimental features. Currently, the following build options are available:
 - `ENABLE_ADDRESS_SANITIZER` and `ENABLE_KERNEL_ADDRESS_SANITIZER`: builds in runtime checks for memory corruption bugs (like buffer overflows and memory leaks) in Lagom test cases and the kernel, respectively.
+- `ENABLE_KERNEL_UBSAN`: builds in runtime checks for detecting undefined behavior in the kernel.
```

Comment:
Nit, feel free to ignore: could we be consistent with the ASan/userspace UBSan flags and call it `ENABLE_KERNEL_UNDEFINED_SANITIZER`.

Reply:
Fixed :)